README.md

```diff
@@ -1,2 +1,29 @@
 # hapi-hpkp
-Hapi module to add HPKP headers to request
+Inspired by [Helmetjs](https://github.com/helmetjs/hpkp), this is a Hapi module to add HPKP headers to all requests.
+
+## Example
+
+```javascript
+var hpkp = require('./index')
+var Hapi = require('hapi')
+
+var hpkpOptions = {
+    maxAge: 1,                       // In seconds
+    sha256s: ["orlando=", "magic="], // Array of sha256
+    includeSubdomains: true,         // optional
+    reportUri: 'http://test.site',   // optional
+    reportOnly: false,               // optional
+}
+
+var server = new Hapi.Server();
```

Comment:
I know it's only example code, but should we be consistent about semi-colons? Either all with or all without I reckon.